code/modules/mob/death.dm

```diff
@@ -41,6 +41,9 @@
 	living_mob_list -= src
 	dead_mob_list += src
 	stat_collection.add_death_stat(src)
+	if(src.client)
+		var/client/C = src.client
```

Comment:
I don't think casting is necessary here, the `client` var already should have the type of `/client` so you can remove this line and just use `client` directly below